pkg/tests/testdata/TestDetailedDiffSet/block_with_computed_and_nested_force_new/added_end.golden

```diff
@@ -52,14 +52,8 @@ Plan: 1 to add, 0 to change, 1 to destroy.
         [urn=urn:pulumi:test::project::crossprovider:index/testRes:TestRes::example]
       ~ tests: [
           ~ [0]: {
```

Comment:
This is an unfortunate side effect of the fact that the engine does not have access to the planned state.
These computed properties are predicted to change but the engine can not display these changes.

We need to figure out a workaround.

Reply:
We've agreed to keep it as is for now and follow up in [#2779](https://github.com/pulumi/pulumi-terraform-bridge/issues/2779)